packages/default/scss/menu/_layout.scss

```diff
@@ -107,6 +110,7 @@
     // Orientation -- vertical
     .k-menu-vertical {
         flex-direction: column;
+        width: 100%;
```

Comment:
In my opinion this _width_ property is not necessary

Reply:
it is necessary as the items shrink in vertical orientation :(

Reply:
Update -> it is needed for the scrollable vertical menu